Tests/Request/ParamFetcherTest.php

```diff
@@ -586,16 +579,12 @@ public function testConstraintThrowExceptionInStrictMode()
             ->method('read')
             ->will($this->returnValue(array('bizoo' => $param)));
 
-        $this->violationFormatter->expects($this->once())
-            ->method('formatList')
-            ->will($this->returnValue('foobar'));
-
         $this->setExpectedException(
             '\\Symfony\\Component\\HttpKernel\\Exception\\BadRequestHttpException',
-            'foobar'
+            "Parameter \"bizoo\" of value \"\" violated a constraint \"expected message 1\"\nParameter \"bizoo\" of value \"\" violated a constraint \"expected message 2\""
```

Comment:
Not sure if this change is right. Should we expect "foobar" instead of the empty string?